root/Readme.ja.md

```diff
@@ -0,0 +1,257 @@
+  <span style="color: red;">*2</span>　リッチクライアント アプリケーションを作成する場合は必須<br />
+  <span style="color: red;">*3</span>　実際のアプリケーションの形態に応じて選択してください
+</div>
+
+上の表を参考に、ビルドバッチファイルを番号順に実行してプログラムをビルドしてください。  
+
+- 必要であれば、環境に合わせて、z_Common.bat 内の BUILDFILEPATH を書き換えてください。  
+
+- ビルド時に以下のエラーが発生した場合は、Windows SDK をインストールしてください。([Open 棟梁の issue](https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/48#issuecomment-241349223) が参考になります。)  
+なお、どのバージョンの Windows 用の Windows SDK を入れたらよいかは、エラーメッセージから判断できます。例えば、以下のメッセージであれば、「"HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\\**v8.0A**\WinSDK-NetFx40Tools-x86" を使用して」とありますので、「Windows 8 用の Windows SDK」をインストールしたらよいことが分かります。
+```
+C:\Windows\Microsoft.NET\Framework\v4.0.30319\Microsoft.Common.targets(2863,5): error MSB3086: タスクは SdkToolsPath "" またはレジストリ キー "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v8.0A\WinSDK-NetFx40Tools-x86" を使用して "AL.exe"を見つけられませんでした。SdkToolsPath が設定されていること、SdkToolsPath の下の適切なプロセッサ固有の場所にツールが存在すること、および Microsoft Windows SDK がインストールされていることを確認してください。 [C:\root\programs\C#\Frameworks\Infrastructure\Public\Public.csproj]
+```
+  
+### ASP.NET 状態サービスの準備
+管理者としてコマンドプロンプトを起動し、下記コマンドを実行します。  
+```bat
+   sc config aspnet_state start= auto
+   net start aspnet_state
+```
+
+### サンプルの実行
+- 以下のファイルを開いてください。
+- web.config または app.config を開き、実際のデータベース環境に合わせて connectionString セクションの値を修正してください。
+- サンプルアプリケーションを実行してください。  
+ログイン画面が出た場合は、任意の英数字を入力してください。(既定ではパスワード認証を行っていません)  
+   
+#### Web の場合：
+C:\root\programs\C#\Samples\WebApp_sample\ProjectX_sample\ProjectX_sample.sln
+ 
+#### C/S 2階層の場合：
+- Windows Forms  
+C:\root\programs\C#\Samples\2CS_sample\2CSClientWin_sample\2CSClientWin_sample.sln
+- WPF  
+C:\root\programs\C#\Samples\2CS_sample\2CSClientWPF_sample\2CSClientWPF_sample.sln
+
+#### C/S 3階層の場合：
+- Windows Forms  
+  - 通常の Windows フォームアプリケーション  
+  C:\root\programs\C#\Samples\WS_sample\WSClient_sample\WSClientWin_sample\WSClientWin_sample.sln
+  - ClickOnce アプリケーション  
+  C:\root\programs\C#\Samples\WS_sample\WSClient_sample\WSClientWinCone_sample\WSClientWinCone_sample.sln
+- WPF  
+C:\root\programs\C#\Samples\WS_sample\WSClient_sample\WSClientWPF_sample\WSClientWPF_sample.sln
+
+### 参考資料
+Open 棟梁をご利用いただくにあたり、OpenTouryoDocument リポジトリのドキュメントをご利用いただけます。
+- [紹介資料](https://github.com/OpenTouryoProject/OpenTouryoDocuments/tree/master/documents/0_Introduction)  
+Open 棟梁の概要資料 (PowerPoint のスライドなど) をご覧いただけます。
+- [利用ガイド](https://github.com/OpenTouryoProject/OpenTouryoDocuments/tree/master/documents/1_User_Guide)  
+Open 棟梁の仕組みや、各機能の仕様などをご覧いただけます。
+- [チュートリアル](https://github.com/OpenTouryoProject/OpenTouryoDocuments/tree/master/documents/2_Tutorial)  
+Open 棟梁のファーストステップガイドです。     
+   
+### 著作権、ライセンス
+[License](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/license)ディレクトリをご確認ください。
+
+### バグ対応
+ご利用いただく中で、バグを発見されましたら、[issue](https://github.com/OpenTouryoProject/OpenTouryo/issues) としてご連絡ください。  
+コミュニティで内容を確認し、適切に対応いたします。
+
+バグ修正パッチは、最新版のテンプレート・ベースを取込むことで適用できます。
```

Comment:
@yusukemaegawa 

There is no line feed in this mark-down.
Line feed code is two space characters in the mark-down.